content/docs/sidebar.json

```diff
@@ -525,5 +525,33 @@
         "slug": "dvclive-with-dvc"
       }
     ]
+  },
+  {
+    "label": "Studio",
+    "slug": "studio",
+    "source": "studio/index.md",
+    "icon": "studio",
+    "children": [
+      {
+        "label": "Get started",
+        "slug": "get-started"
+      },
+      {
+        "label": "Create a view",
+        "slug": "create-view"
+      },
+      {
+        "label": "Visualize experiments",
```

Comment:
```suggestion
        "label": "Visualize Experiments",
```

Reply:
Should this one go after "Run Experiments" BTW? Creating experiments seems to come before seeing them but I'm not sure if that's true for Studio.

Reply:
> in most cases there will already be experiments when the view is created

Right, this is a pre-requisite (to my point).

> It will probably be easier for users to understand if they first see a visualization of the experiments

Maybe. I need to check the actual docs contents to have an opinion on this. Let's keep this open for now.